packages/react-native/bugsnag-react-native-xcode.sh

```diff
@@ -41,8 +41,13 @@ ARGS=(
 
 case "$CONFIGURATION" in
   *Debug*)
-    ARGS+="--dev"
+    ARGS+=("--dev")
     ;;
 esac
 
-../node_modules/.bin/bugsnag-source-maps upload-react-native "${ARGS[@]}"
\ No newline at end of file
+if [ ! -z "$ENDPOINT" ]; then
+  ARGS+=("--endpoint")
+  ARGS+=("$ENDPOINT/sourcemaps")
```

Comment:
Just confirmed with the pipeline team, this should be singular:

```suggestion
  ARGS+=("$ENDPOINT/sourcemap")
```

Also just wondering if there's a straightforward way to ensure there's only 1 `/`, if the provided endpoint has a trailing slash?